addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2361,8 +2361,9 @@ const Wysiwyg = Widget.extend({
     },
     _onSelectionChange() {
         if (this.odooEditor.autohideToolbar) {
+            const selectionInLink = document.querySelector('span[data-o-link-zws="end"]');
```

Comment:
This constant don't ensure the selection is in a link, it only ensure we are in a link.
this create issue when two  link are side to side ( see screenshot) 
![image](https://github.com/odoo/odoo/assets/3356461/f9a9899d-d4bc-43df-82b2-7b4e29eccf57)

Reply:
@visp-odoo I'd like to have a better way to tell if the selection is in the correct link.

Somethink like :

```javascript
const selectionInLink = odooEditor.getSelection().startElement === this.linkPopover.target;
if (isVisible && !selectionInLink ) {
 ....
```
( this is an example on principle not an actual working solution )

It would be much cleaner. Could you research the best way to do that please.